src/main/java/server/DeleteRequestHandler.java

```diff
@@ -0,0 +1,27 @@
+package server;
+
+import HTTPComponents.StatusCode;
+
+import java.io.File;
+
+public class DeleteRequestHandler implements RequestHandler {
+
+    public Response handle(Request request) {
+        if (delete(request.getResourceIdentifier())) {
+            return new ResponseBuilder()
+                    .setStatusCode(StatusCode.NO_CONTENT)
+                    .build();
+        } else {
+            return new ResponseBuilder()
+                    .setStatusCode(StatusCode.NOT_ALLOWED)
+                    .build();
+        }
+    }
+
+    private boolean delete(String resourceIdentifier) {
+        String fullPath = new File("src/main/resources/" + resourceIdentifier).getAbsolutePath();
+        File resource = new File(fullPath);
+        return resource.delete();
```

Comment:
what happens if this fails?

Reply:
Right now, if it succeeds in deleting the file, it returns true and the response is 204 No Content. If it fails and returns false, it returns 405 Method Not Allowed. That seemed the closest, but I'm not sure what the best way to give more useful information to the user is. It should at least log something here - punting that part to the logger refactor.